src/models/Review.py

```diff
@@ -0,0 +1,36 @@
+#temp
+from bson import ObjectId
+import datetime
+
+class Review:
+  _id: ObjectId
```

Comment:
Sharing what I learned:

Originally I thought this were instance-level attributes, but after further researching I've learned that I was wrong. This represent _class-level_ attributes so the end result would've been all user sharing the same handful of vars. The solution is just doing things the old-fashioned way in `__init__()`. All assignments and declarations of object attributes should happen there.